include/d/actor/d_a_deku_item.h

```diff
@@ -20,8 +33,28 @@ class daDekuItem_c : public fopAc_ac_c {
     void checkOrder();
     BOOL _draw();
 
+    static const char m_arcname[];
+
 public:
-    /* Place member variables here */
+    /* 0x290 */ request_of_phase_process_class mPhs; /* inferred */
```

Comment:
the `/* inferred */` comments aren't necessary.

also if you want a way of exporting structs from ghidra that doesn't add these, there's a ghidra script pinned in the discord